src/views/Profile/AuthorsList/AuthorListDropdown.tsx

```diff
@@ -3,9 +3,9 @@ import { FormControl, Select, InputBase, MenuItem } from '@material-ui/core';
 import { useStyles } from './styles/AuthorListDropdown.styles';
 
 interface IAuthorListDropdownProps {
-  pageSizes: number[];
-  selected: number;
-  setChanges: (payload: any) => void;
+  pageSizes: Array<number | string>;
```

Comment:
I believe we should strict: `pageSizes: Array<number | 'All'>`; similarly to `selected` property.

Reply:
We have problem like this
![image](https://user-images.githubusercontent.com/62579471/220009388-bd961083-3eb8-4f51-8978-9d83d418a1fd.png)

Reply:
give me some time, I'll play with it